spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberTemplate.java

```diff
@@ -111,9 +112,7 @@ public Subscriber subscribe(String subscription, MessageReceiver messageReceiver
 
 		List<AcknowledgeablePubsubMessage> ackableMessages = pull(pullRequest);
 
-		this.ack(ackableMessages.stream()
-				.map(AcknowledgeablePubsubMessage::getAckId)
-				.collect(Collectors.toList()), pullRequest.getSubscription());
+		this.ack(ackableMessages);
```

Comment:
You still use `this.` here. 
This is kinda not our code style and the problem that we can't enforce it via Checkstyle rule...